src/components/Table/views/Tree/index.tsx

```diff
@@ -351,45 +364,36 @@ class TreeView extends Component<Props, State> {
                   },
                 }}
               >
-                {currentNode?.id}: {currentNode?.name}
+                {currentNode?.name}
               </Link>
             </header>
             {currentNode?.rank?.toLowerCase() !== 'no rank' && (
               <div>
-                <Tooltip title="Rank.">
-                  <span
-                    className={css('small', 'icon', 'icon-common')}
-                    data-icon="&#xf129;"
-                    aria-label="Rank."
-                  />
-                </Tooltip>{' '}
-                <i>{currentNode?.rank}</i>
+                <p>{currentNode?.rank}</p>
               </div>
             )}
             {currentNode?.lineage && (
-              <DropDownButton label="Lineage" fontSize="12px">
-                <ul>
-                  {Array.from(this._lineageNames.keys()).map((key) => (
-                    <li key={key}>
-                      <Link
-                        to={{
-                          description: {
-                            main: { key: 'taxonomy' },
-                            taxonomy: { db: 'uniprot', accession: key },
-                          },
-                        }}
-                      >
-                        {`${
-                          this._lineageNames
-                            ?.get(key)
-                            ?.charAt(0)
-                            ?.toUpperCase() || ''
-                        }${this._lineageNames.get(key)?.slice(1) || ''}`}
-                      </Link>
-                    </li>
-                  ))}
-                </ul>
-              </DropDownButton>
+              <nav className={'breadcrumbs'}>
```

Comment:
```suggestion
              <nav className={css('breadcrumbs')}>
```

Then add the following to `src/components/Table/views/Tree/style.css`:

```css
.breadcrumbs > li {
  text-transform: none !important;
}
```